scripts/setup/gen_pigweed_cipd.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+import json
+
+_LIST_OF_PACKAGES_TO_EXCLUDE = {'rust'}
```

Comment:
Arguably I might want to provide `fuchsia/third_party/rust/` here instead, and then see if the package begins with that string instead of doing the string split below.